vue/components/ui/button-color-platforme/button-color-platforme.vue

```diff
@@ -22,7 +22,7 @@
     cursor: pointer;
     display: inline-block;
     font-size: 13px;
-    font-weight: 600;

```

Comment:
Let's leave this as it was, even if it's different from Zeplin

Reply:
Now, all the button use font-weight: bold. I defined globally